eng/pipelines/libraries/stress/ssl.yml

```diff
@@ -72,5 +81,14 @@ jobs:
   
   - powershell: |
       cd '$(sslStressProject)'
+      $env:SSLSTRESS_CLIENT_ARGS = "$env:SSLSTRESS_CLIENT_ARGS -http 2.0"
+      $env:SSLSTRESS_SERVER_ARGS = "$env:SSLSTRESS_SERVER_ARGS -http 2.0"
+      docker-compose up --abort-on-container-exit --no-color
+    displayName: Run SslStress - HTTP/2
```

Comment:
`HTTP/2` while https://github.com/dotnet/runtime/pull/41592/files#diff-53dfbdf75e79a04b57ab654be70b8cfcR86 has `HTTP 2.0` and all 1.1 have `HTTP 1.1`.